src/Components/Buttons.scss

```diff
@@ -1,3 +1,5 @@
 .btn-main {
-    background-color: #077D7F;
-}
\ No newline at end of file
+    background-color: #077D7F !important;
+    margin: 0.5rem;
+    color: white !important
```

Comment:
@valeriecollinsdesign Was the goal was to have the text color be yellow? That got lost in the second commit and it got put back to white here.

Reply:
Yes, that was. I went back and hopefully reupdated it and send it another pull request.